### ISSUE TITLE
Metabox priority breaks Xtreme Builder theme metabox on Post Edit screen

### ISSUE THREAD

**Issue by arnaudbroes** [semperfiwebdesign/all-in-one-seo-pack#1605]
Reported by Jeroen Tabbernee on the Premium Forum here - https://semperplugins.com/support/troubleshooting-all-in-one-seo-pack-pro/after-upgrading-last-version-of-aio-seo-pro-the-xtreme-theme-cant-be-modified/#p7565

I troubleshooted this issue and found out that this broke with the release of 2.3.16 (2.4.16 for Pro), specifically by this commit - https://github.com/semperfiwebdesign/all-in-one-seo-pack/commit/571ea2b11cd7277eaa10ce4982610d2e31cb84a0. Contact me for the theme file.

**Comment by EkoJr** [comment]
I believe all that would be needed in this case is a filter hook function to be added to the site.
```
function aioseop_metabox_priority( $priority ) {  
  return 'low';  
}  
add_filter( 'aioseop_post_metabox_priority', 'aioseop_metabox_priority' );
```
There seems to be a number of issues, and also seems to require PHP 5.X to run the theme. Although, the main issue is occuring because of the metabox priority being set to high, and a JS function `window.postboxes.save_order(pagenow)`  (within xtreme) which primarily triggers it.

This may be something fixed in a future release of the theme/plugin, but can be avoided by using AIOSEOP's filter hook.

**Comment by michaeltorbert** [comment]
@EkoJr Is it something the theme is doing wrong or something we're doing wrong?

**Comment by EkoJr** [comment]
@michaeltorbert It looks like it would be something the theme is doing wrong.

I noticed similar issues being reported to other plugins, and it seems to be a conflict with the 'top' metabox (aka priority = high). Which would work normally as intended, until the JS function, `window.postboxes.save_order(pagenow)`, is used within the theme.

So I tested a couple other plugins with metaboxes to see if they had the same issue, and it looks like it would be focused around the high priority setting for the metaboxes. 

The changes required seemed to be with the theme, but luckily there was a filter hook added with AIOSEOP for a workaround.
